.releaserc

```diff
@@ -1,5 +1,19 @@
 {
   "branches": [
     "master"
+  ],
+  "plugins": [
+    "@semantic-release/commit-analyzer",
+    "@semantic-release/release-notes-generator",
+    "@semantic-release/changelog",
+    "@semantic-release/npm",
+    "@semantic-release/github",
+    [
+      "@semantic-release/git",
+      {
+        "assets": ["CHANGELOG.md", "package.json"],
+        "message": "chore(release): ${nextRelease.version} [skip ci]"
```

Comment:
GitHub Actions does not understand `skip ci` by default. We need to manually check for it. You can have a look at how it was done in react-native-test-app: https://github.com/microsoft/react-native-test-app/commit/c20eaaaba93e0af687648def954ff6e65f36194a

Reply:
Never mind. It looks like it was added earlier this year: https://github.blog/changelog/2021-02-08-github-actions-skip-pull-request-and-push-workflows-with-skip-ci/

Reply:
good one, will apply that